test/support/generator/get-geospatial-addresses-generator.ts

```diff
@@ -0,0 +1,230 @@
+import { ENUMS, GEOSPATIAL } from '@ukef/constants';
+import { GetAddressOrdnanceSurveyResponse } from '@ukef/helper-modules/ordnance-survey/dto/get-addresses-ordnance-survey-response.dto';
+import { OrdnanceSurveyAuthErrorResponse } from '@ukef/helper-modules/ordnance-survey/dto/ordnance-survey-auth-error-response.dto';
+import { GetAddressByPostcodeQueryDto } from '@ukef/modules/geospatial/dto/get-address-by-postcode-query.dto';
+import { GetAddressesResponse } from '@ukef/modules/geospatial/dto/get-addresses-response.dto';
+
+import { AbstractGenerator } from './abstract-generator';
+import { RandomValueGenerator } from './random-value-generator';
+
+export class GetGeospatialAddressesGenerator extends AbstractGenerator<AddressValues, GenerateResult, GenerateOptions> {
+  constructor(protected readonly valueGenerator: RandomValueGenerator) {
+    super(valueGenerator);
+  }
+
+  protected generateValues(): AddressValues {
+    return {
+      ORGANISATION_NAME: this.valueGenerator.word({ length: 5 }),
+      BUILDING_NAME: this.valueGenerator.word(),
+      BUILDING_NUMBER: this.valueGenerator.nonnegativeInteger().toString(),
+      THOROUGHFARE_NAME: this.valueGenerator.word({ length: 7 }),
```

Comment:
It can, but I wanted to generate 5 words for address to look better, but generated 5 letter word by mistake.

I am changing `.word({length: 7})` to `.sentence({words: 5})` , I hope it makes sense.